pkgs/applications/misc/safeeyes/default.nix

```diff
@@ -16,59 +29,55 @@ in buildPythonApplication rec {
   buildInputs = [
     gtk3
     gobject-introspection
```

Comment:
This must go to nativeBuildInputs for the setup hook to work.

Reply:
I tried that but there is no difference.